packages/firebase/index.d.ts

```diff
@@ -5472,6 +5472,52 @@ declare namespace firebase.storage {
      *     including if the object did not exist.
      */
     updateMetadata(metadata: firebase.storage.SettableMetadata): Promise<any>;
+    /**
+     * List items and folders (prefixes) within this directory.
+     * "/" is treated as a path delimiter. Firebase storage does not support invalid object path that ends with "/" or contains two consecutive "//".
+     * All invalid objects in GCS will be filtered.
+     *
+     * @param {!options.maxResults} If set, limits the total number of prefixes and items to return..
+     * @param {!options.pageToken} The nextPageToken from a previous list() response. If provided, listing is resumed from that position.
+     * @return {!firebase.Promise<firebase.storage.ListResult>} A promise that resolves with the sub-directories and items.
```

Comment:
@Feiyang1 Should we drop the return types? And if not, does it make sense to use a standard `Promise` here, since that is what we are returning?